ROADMAP.md

```diff
@@ -0,0 +1,14 @@
+# Spin Roadmap
+
+Spin is an open-source framework designed to facilitate the development and deployment of event-driven, serverless applications using WebAssembly (Wasm). Spin provides a lightweight runtime environment that encompasses Wasmtime for running fast and secure applications integrating the latest standards in the WebAssembly community. We’re excited for the growing engagement in the Spin community and for the hard earned continued development and progress in the WebAssembly community that are incorporated in the Spin project.
+
+Taking these considerations into account and the feedback for the community, the Spin project is focusing on features and improvements both as a developer tool and as an extensible runtime environment. The following is a non-exhaustive list:
+
+1. Composition and polyglot development experience
```

Comment:
The rendering doesn't make it clear that there is a subject to each point. Maybe add a hyphen or colon
```suggestion
1. Composition and polyglot development experience -
```